src/sidebar/store/modules/filters.ts

```diff
@@ -20,73 +23,62 @@ import { createStoreModule, makeAction } from '../create-store';
  *   (see `util/search-filter`)
  */
 
-/**
- * @typedef {import('../../../types/config').FocusConfig} FocusConfig
- * @typedef {import('../../../types/config').SidebarSettings} SidebarSettings
- * @typedef {import('../../../types/rpc').FocusUserInfo} FocusUserInfo
- */
-
-/**
- * @typedef FilterOption
- * @prop {string} value - The machine-readable value of the option
- * @prop {string} display - The human-facing "pretty" value of the option
- */
+export type FilterOption = {
+  /** The machine-readable value of the option */
+  value: string;
+  /** The human-facing "pretty" value of the option */
+  display: string;
+};
 
 /**
  * Valid/recognized filters
- * @typedef {'user'} FilterKey
  */
+type FilterKey = 'user';
 
-/**
- * @typedef {Record<FilterKey, FilterOption|undefined>} Filters
- */
+type Filters = Partial<Record<FilterKey, FilterOption | undefined>>;
```

Comment:
This type is referenced in exported types, so I think it makes sense to export as well.